agent/command/exec_test.go

```diff
@@ -187,7 +187,10 @@ func (s *execCmdSuite) TestRunCommandPropgatesError() {
 	cmd.SetJasperManager(s.jasper)
 	s.NoError(cmd.ParseParams(map[string]interface{}{}))
 	exec := cmd.getProc(s.ctx, "foo", s.logger)
-	s.Error(cmd.runCommand(s.ctx, "foo", exec, s.logger))
+	err := cmd.runCommand(s.ctx, "foo", exec, s.logger)
```

Comment:
This is unrelated to this PR, but it's gonna bother me...

Can you fix the spelling for `Propagates` on L183?

Reply:
😂 done